practice.py

```diff
@@ -218,10 +223,10 @@ def __init__(self, verbs):
 
                 def question(self, verb: str, answer: str) -> bool:
                     """Check knowledge of a translation"""
-                    print('\nVerb:', verb)
+                    print(f'\n{verb}')
                     input()
```

Comment:
Unite it into one: input(f'\n{verb}') to remove extra empty line which now looks too empty, I think.